unit_tests/test_tlslite_extensions.py

```diff
@@ -1679,5 +1680,53 @@ def test_parse_with_trailing_data(self):
         with self.assertRaises(SyntaxError):
             self.ext.parse(parser)
 
+class TestHeartbeatExtension(unittest.TestCase):
+    def test___init___(self):
+        ext = HeartbeatExtension()
+
+        self.assertIsNotNone(ext)
+        self.assertEqual(ext.extType, 15)
+        self.assertIsNone(ext.mode)
+
+    def test_create(self):
+        ext = HeartbeatExtension().create(0)
+
+        self.assertIsNotNone(ext)
+        self.assertEqual(ext.extType, 15)
+        self.assertEqual(ext.mode, 0)
+
+    def test_extData_none_mode(self):
+        ext = HeartbeatExtension()
+
+        self.assertEqual(ext.extData, bytearray(0))
+
+    def test_extData_mode(self):
+        ext = HeartbeatExtension().create(0)
+
+        self.assertEqual(ext.extData, b'\x00')
+
+    def test_parse_with_no_data(self):
+        parser = Parser(bytearray(0))
+
+        ext = HeartbeatExtension()
+
+        with self.assertRaises(SyntaxError):
+            ext.parse(parser)
+
+    def test_parse(self):
+        parser = Parser(bytearray(b'\x00'))
+
+        ext = HeartbeatExtension().parse(parser)
+
+        self.assertEqual(0, ext.mode)
+
+    def test_parse_with_too_much_data(self):
+        parser = Parser(bytearray(b'\x00\x00'))
+
+        ext = HeartbeatExtension()
+
+        with self.assertRaises(SyntaxError):
+            ext.parse(parser)
+
```

Comment:
I think that you meant test for `createResponse`, right? It is at `unit_tests/test_tlslite_messages.py`.